devops/dependencies.json

```diff
@@ -18,6 +18,12 @@
       "url": "https://github.com/intel/cm-compiler/releases/tag/cmclang-1.0.144",
       "root": "{DEPS_ROOT}/opencl/runtime/linux/oclgpu"
     },
+    "level_zero": {
+      "github_tag": "v1.8.8",
+      "version": "v1.8.8",
+      "url": "https://github.com/oneapi-src/level-zero/releases/tag/v1.8.8",
```

Comment:
This URL points to Level Zero loader and I think the runtime is installed as part of compute-runtime package. Isn't loader installed with compute-runtime as well?
If not, I'm okay with installing the loader, but PR title should be corrected.

Reply:
Loader is not installed as part of the compute-runtime.
Here is content for the record. Here is what is installed with compute-runtime package:

```
dpkg-query -L intel-level-zero-gpu
/.                                                        
/usr                                                      
/usr/include                                              
/usr/lib                                                  
/usr/lib/x86_64-linux-gnu                                 
/usr/lib/x86_64-linux-gnu/libze_intel_gpu.so.1.3.24278    
/usr/share                                                
/usr/share/doc                                            
/usr/share/doc/intel-level-zero-gpu                       
/usr/share/doc/intel-level-zero-gpu/changelog.gz          
/usr/share/doc/intel-level-zero-gpu/copyright             
/var                                                      
/var/lib                                                  
/var/lib/libze_intel_gpu                                  
/var/lib/libze_intel_gpu/pci_bind_status_file             
/var/lib/libze_intel_gpu/wedged_file                      
/usr/lib/x86_64-linux-gnu/libze_intel_gpu.so.1
```
And here is what this PR adds:
```
dpkg-query -L level-zero                       
/usr                                                                   
/usr/lib                                                               
/usr/lib/x86_64-linux-gnu                                              
/usr/lib/x86_64-linux-gnu/libze_loader.so.1.8.8                        
/usr/lib/x86_64-linux-gnu/libze_tracing_layer.so.1.8.8                 
/usr/lib/x86_64-linux-gnu/libze_validation_layer.so.1.8.8              
/usr/lib/x86_64-linux-gnu/libze_loader.so.1                            
/usr/lib/x86_64-linux-gnu/libze_tracing_layer.so.1                     
/usr/lib/x86_64-linux-gnu/libze_validation_layer.so.1                  

dpkg-query -L level-zero-devel                 
/usr                                                                   
/usr/include                                                           
/usr/include/level_zero                                                
/usr/include/level_zero/layers                                         
/usr/include/level_zero/layers/zel_tracing_api.h                       
/usr/include/level_zero/layers/zel_tracing_ddi.h                       
/usr/include/level_zero/layers/zel_tracing_register_cb.h               
/usr/include/level_zero/loader                                         
/usr/include/level_zero/loader/ze_loader.h                             
/usr/include/level_zero/ze_api.h                                       
/usr/include/level_zero/ze_ddi.h                                       
/usr/include/level_zero/zes_api.h                                      
/usr/include/level_zero/zes_ddi.h                                      
/usr/include/level_zero/zet_api.h                                      
/usr/include/level_zero/zet_ddi.h                                      
/usr/lib                                                               
/usr/lib/x86_64-linux-gnu                                              
/usr/lib/x86_64-linux-gnu/pkgconfig                                    
/usr/lib/x86_64-linux-gnu/pkgconfig/level-zero.pc                      
/usr/lib/x86_64-linux-gnu/pkgconfig/libze_loader.pc                    
/usr/lib/x86_64-linux-gnu/libze_loader.so                              
/usr/lib/x86_64-linux-gnu/libze_tracing_layer.so                       
/usr/lib/x86_64-linux-gnu/libze_validation_layer.so                  
```